### ISSUE TITLE
Is there a method to understand if a person has removed the miner in the PC?

### ISSUE THREAD

**Issue by Frax-Python** [UnamSanctam/UnamWebPanel#234]

**Comment by UnamSanctam** [comment]
Not really, not unless you also have a RAT on it or something. It's common to assume that it has been removed after being offline for 3+ days.

**Comment by Frax-Python** [comment]
can u made for me a RAT?

**Comment by UnamSanctam** [comment]
A RAT is a Remote Administration Tool, it's a program that gives control over a computer. There are many online. If you were to use a RAT then it would be better to spread that and then upload the miner to the computers through the RAT instead.

**Comment by PeakyPinker** [comment]
> A RAT is a Remote Administration Tool, it's a program that gives control over a computer. There are many online. If you were to use a RAT then it would be better to spread that and then upload the miner to the computers through the RAT instead.

can you btc clipper builder source code  ?

**Comment by TomerGamerTV** [comment]
> > A RAT is a Remote Administration Tool, it's a program that gives control over a computer. There are many online. If you were to use a RAT then it would be better to spread that and then upload the miner to the computers through the RAT instead.
> 
> can you code btc clipper builder source code ?

there are many builders inside github just search
